codeigniter4/CodeIgniter4
# feat: Add Strict Validation

**Description**
Supersedes #5375
Fixes #5374
Fixes #5368

- add `CodeIgniter\ValidationStrict`
  - add `declare(strict_types=1)` for Validation and Rule classes
  - remove parameter types for data to validate in validation rules
  - separate validation process of multiple field and single field

See #5375, if you want to see the change to the current Validation class.

**Checklist:**
